android/src/main/kotlin/org/dasfoo/flutter_sentry/FlutterSentryPlugin.kt

```diff
@@ -71,6 +72,8 @@ class FlutterSentryPlugin : FlutterPlugin, MethodCallHandler {
         } else if (call.method == "getFirebaseTestLab") {
             result.success(firebaseTestLab)
             return
+        } else if (call.method == "setEnvironment") {
+            setEnvironment(call, result)
```

Comment:
I think this branch is missing a `return` statement: it will otherwise fallthrough into `result.notImplemented` below.

Reply:
Good call. Updated.